### DISCUSSION TITLE
End point for Supply and Demand

### DISCUSSION THREAD

**Discussion by philipmat** [ercot/api-specs#54] [Q&A] [answered]
Is there an end-point to produce the data that powers the [Supply and Demand dashboard](https://www.ercot.com/gridmktinfo/dashboards/supplyanddemand) ?

Is not, is there an ETA for such an end-point? 

Thanks!

**Comment by mackermann-ercot** [comment, marked as answer]
Hello Philip,

Currently, there are no plans to reproduce the [Supply and Demand](https://www.ercot.com/gridmktinfo/dashboards/supplyanddemand) data directly as an API endpoint.

Regards,
-mackermann, ERCOT

**Comment by victorthemtt** [comment]
Hello Philip, 

Try sending a GET request to this url "https://www.ercot.com/api/1/services/read/dashboards/supply-demand.json". It's the same as an endpoint request. 

Victor Mao
